### PR TITLE
Add enzyme

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8566/

File: jujugui/static/gui/src/test/enzyme-setup.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Enzyme = require('enzyme');
+const Adapter = require('enzyme-adapter-react-16');
+// const chai = require('chai');
```
Comment (hatched) [inline jujugui/static/gui/src/test/enzyme-setup.js]: commented out?

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8567/

**Comment by huwshimi** [comment]
[Add Enzyme](https://trello.com/c/VIquDYVD/386-add-enzyme)

**Pull request merged**